packages/dokeysto_camltc/dokeysto_camltc.3.0.2/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/dokeysto"
+bug-reports: "https://github.com/UnixJunkie/dokeysto/issues"
+dev-repo: "git+https://github.com/UnixJunkie/dokeysto.git"
+license: "LGPL-2.1"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  [make "test_camltc"] {with-test}
+]
+depends: [
+  "ocaml"
+  "dune" {>= "1.0"}
+  "dokeysto"
```

Comment:
```suggestion
  "dokeysto" {= version}
```
Should they also match their current version then?

Reply:
No, that's not necessary.